.github/workflows/test.yml

```diff
@@ -1,29 +1,26 @@
-on: [push, pull_request]
-name: test
+name: Test
+
+on:
```

Comment:
https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/2

Reply:
that solves one of the mysteries of gh-actions for me. thanks